.github/workflows/enzyme-ci.yml

```
@@ -49,7 +49,7 @@ jobs:
           mkdir -p build
           cd build
           rm -f config.toml
-          ../configure --enable-llvm-link-shared --enable-llvm-plugins --enable-llvm-enzyme --release-channel=nightly --enable-llvm-assertions --enable-lld --enable-option-checking --enable-ninja --disable-docs
+          ../configure --enable-llvm-link-shared --enable-llvm-enzyme --use-lld=\"self-contained\" --release-channel=nightly --enable-llvm-assertions --enable-option-checking --enable-ninja --disable-docs
```

Let's try deleting it.
```suggestion
          ../configure --enable-llvm-link-shared --enable-llvm-enzyme --release-channel=nightly --enable-llvm-assertions --enable-option-checking --enable-ninja --disable-docs
```

---

That will work, but then it will fall back to something slower and increase link time. Let's at least use a global lld which we can make available in the runner (though rust-lld is preferable).

---

ok, rg in src/bootstrap/src/core/config/config.rs gives:
```
src/bootstrap/src/core/config/config.rs
114:/// - Self-contained: `-Clinker-flavor=gnu-lld-cc -Clink-self-contained=+linker`
1035:                formatter.write_str("one of true, 'self-contained' or 'external'")
1051:                    "self-contained" => Ok(LldMode::SelfContained),
1634:                    "Trying to use self-contained lld as a linker, but LLD is not being added to the sysroot. Enable it with rust.lld = true."
```
So self-contained probably will enforce building lld even if we can figure out how to use it.
Let's just add lld to the instance and simply use true, to use it.